testdata/TestE2E-expected-node_gen.go

```diff
@@ -10,6 +10,13 @@ func NodeToCore(s *Node, t *core.ClusterNode) {
 	}
 	t.ID = s.ID
 	t.Label = core.Label(s.Label)
+	t.Flag = &s.Flag
```

Comment:
This shows you what the updated basic type conversion code looks like when crossing pointer/nonpointer boundaries.